src/mbgl/style/sources/raster_dem_source.cpp

```diff
@@ -1,18 +1,77 @@
+#include <mbgl/storage/file_source.hpp>
 #include <mbgl/style/conversion/json.hpp>
 #include <mbgl/style/conversion/tileset.hpp>
 #include <mbgl/style/layer.hpp>
 #include <mbgl/style/source_observer.hpp>
 #include <mbgl/style/sources/raster_dem_source.hpp>
 #include <mbgl/style/sources/raster_source_impl.hpp>
 #include <mbgl/tile/tile.hpp>
+#include <mbgl/util/async_request.hpp>
+#include <mbgl/util/exception.hpp>
 #include <mbgl/util/mapbox.hpp>
 #include <utility>
 
 namespace mbgl {
 namespace style {
 
-RasterDEMSource::RasterDEMSource(std::string id, variant<std::string, Tileset> urlOrTileset_, uint16_t tileSize)
-    : RasterSource(std::move(id), std::move(urlOrTileset_), tileSize, SourceType::RasterDEM) {}
+RasterDEMSource::RasterDEMSource(std::string id,
+                                 variant<std::string, Tileset> urlOrTileset_,
+                                 uint16_t tileSize,
+                                 std::optional<Tileset::DEMEncoding> encoding)
+    : RasterSource(std::move(id), std::move(urlOrTileset_), tileSize, SourceType::RasterDEM),
+      encoding(encoding) {}
+
+void RasterDEMSource::loadDescription(FileSource& fileSource) {
```

Comment:
I don't see any difference with `RasterSource::loadDescription`. Why did you override it?